airbyte-integrations/connectors/source-shopify/source_shopify/schemas/metafields.json

```diff
@@ -29,13 +29,22 @@
       "type": ["null", "string"]
     },
     "value": {
-      "type": ["null", "integer", "object", "string"],
-      "properties": {}
+      "oneOf": [
+        {
```

Comment:
in this case we should just make this a string since most destination warehouses don't support `oneOf`s

Reply:
What if we have something else rather than the actual `string` coming from the source?

Reply:
since all these types can be converted to string, it is preferable to use it. For example redshift doesn't have a concept of a column which is either a boolean OR an int OR a string. So we just use string :)